source/element-types.md

```diff
@@ -1,4 +1,227 @@
 (c:ele.types)=
 # Element Types
 
-Under construction...
\ No newline at end of file
+%---------------------------------------------------------------------------------------------------
+(s:ackicker)=
+## ACKicker Element
+
+Time varying kicker element
+
+Under Construction...
+
+%---------------------------------------------------------------------------------------------------
+(s:beambeam)=
+## BeamBeam Element
+
+Colliding beam element.
```

Comment:
Change to "Crossing beams element."?

I also wonder if "CrossingBeams Element" would not be more descriptive than "BeamBeam Element".

Reply:
Fixed. Changed to "Element for simulating colliding beams."